### DISCUSSION TITLE
Example of “Periodic Script”

### DISCUSSION THREAD

**Discussion by BrighamEaquinto** [quarto-dev/quarto-cli#8452] [Q&A]
Does anyone have an example of using periodic scripts in a quarto project per this page? https://quarto.org/docs/projects/scripts.html

Specifically, I'm wondering if I can have a script that pulls data from a database and save that data in a directory in the project run nightly. This way, time is saved because the query wouldn't run every time the user needs at the dashboard, but rather be able to use the data pulled from the previous night that's already saved in the project. 

Thanks all

**Comment by mcanouil** [comment]
There is no limit of what the script can do (as long as it is Lua, TypeScript, R, or Python when using `quarto run`).
You just have to write it, ensure you have the required dependencies for your script.

So, write your script with the code needed to pull the data and save the data.
Put the whole thing in a CI workflow or cron table to schedule it and here you go.
